src/ripple/app/consensus/RCLConsensus.cpp

```diff
@@ -551,18 +551,19 @@ RCLConsensus::Adaptor::doAccept(
         // in the previous consensus round.
         //
         bool anyDisputes = false;
-        for (auto& it : result.disputes)
+        for (auto& [_, dispute] : result.disputes)
```

Comment:
auto const&

Reply:
Fixed in a225b62e5 `Add const to some structured bindings`